hasher/README.md

```diff
@@ -1,20 +1,64 @@
 # Hasher API
 
+_The secure hashing service_.
 
-_The secure hashing service_
+This package provides a _FastAPI_ service that can be used to generate secure hashes.
+It is used by the [PIXL EHR API](../pixl_ehr/README.md) (for EHR anonymisation) and [PIXL Orthanc
+Anon](../orthanc_anon/README.md) (for DICOM image anonymisation) services.
+
+## Local development
+
+### Dependencies
+
+It is assumed you have a Python virtual environment configured using a tool like Conda or pyenv.
+Install the dependencies from inside the _PIXL/hasher/src_ directory:
+
+```bash
+pip install -e .
+```
+
+### Setup
+
+Create a _local.env_ file in _PIXL.hasher/src/hasher_ from _local.env.sample_ in the same location.
+Use the credentials stored in the `Hasher API dev secrets` note in LastPass to populate the
+environment variables.
+Set `LOG_ROOT_DIR` to anywhere convenient.
+
+### Run
+
+from the _PIXL/hasher/src_ directory:
+
+```bash
+uvicorn hasher.main:app --host=0.0.0.0 --port=8000 --reload
+```
+
+### Test
+
+From this directory:
+
+```bash
+pytest
+```
+
+to skip linting and run only the last failed test.
```

Comment:
Does this run only the last failed test?

Reply:
I don't think it does, no idea why that's mentioned there 🤷‍♂️